Sources/MongoSwift/MongoClient.swift

```diff
@@ -92,15 +96,15 @@ public struct ClientOptions: CodingStrategyProvider, Decodable {
 /// Options to use when retrieving a `MongoDatabase` from a `MongoClient`.
 public struct DatabaseOptions: CodingStrategyProvider {
     /// A read concern to set on the retrieved database. If one is not specified, the database will inherit the
-    /// client's read concern.
+    /// client's read concern. To set the read concern to the server's default, specify an empty ReadConcern.
```

Comment:
Similar to my other comment, I think we can consolidate these various repeated sentences to the options structs' docstrings themselves. Also, we want to be emphasizing that a user would only need to do this if a non-default option was set somewhere up the chain, so we could say "To override a option inherited from the parent client (e.g. a read preference) with the default value for that option, it must be explicitly specified (e.g. `ReadConcern()`, not `nil`).", or something along those lines. Ditto for `CollectionOptions`.